web/src/docker/Dockerfile

```diff
@@ -4,14 +4,15 @@ ENV XMS=1G XMX=6G
 
 RUN java -jar "$JETTY_HOME/start.jar" --create-startd --add-to-start=jmx,jmx-remote,stats
 
-ADD . /
+ADD --chown=jetty:jetty ./var/lib/jetty/webapps /var/lib/jetty/
```

Comment:
FYI, this copies the contents of the webapps dir into jetty, which is not what is expected.

Reply:
Fixed with ed77e9b8f33599e4d3ae047424942c8580843d79